x-pack/packages/kbn-elastic-assistant/impl/assistant/api/anonymization_fields/use_bulk_anonymization_fields.ts

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { HttpSetup, IToasts } from '@kbn/core/public';
+import {
+  ELASTIC_AI_ASSISTANT_API_CURRENT_VERSION,
+  ELASTIC_AI_ASSISTANT_ANONYMIZATION_FIELDS_URL_BULK_ACTION,
+} from '@kbn/elastic-assistant-common';
+import {
+  PerformBulkActionRequestBody,
+  PerformBulkActionResponse,
+} from '@kbn/elastic-assistant-common/impl/schemas/anonymization_fields/bulk_crud_anonymization_fields_route.gen';
+
+export const bulkChangeAnonymizationFields = async (
```

Comment:
```suggestion
export const bulkUpdateAnonymizationFields = async (
```

Can we rename this and the similarly named `bulkChangeConversations` to `bulkUpdateConversations`

Additionally, since this is not a hook, can we rename the file to not have the word `use_`? `bulk_update_anonymization_fields.ts`

Reply:
Maybe you want to keep `Change` but can you definitely remove the word `use_` from the file?